### ISSUE TITLE
[Content Author Module 6: Multimedia] Government regulations and other policies user needs vs minimal requirements

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-curricula#560]
> “Explain that different types of audio and video content have different accessibility requirements based on governmental regulations and other policy requirements.”

Well, I’d say they have the same requirements to be fully accessible!; however, some ‘governmental regulations and other policy requirements’ don’t actually require full accessibility. :-) 

In <https://www.w3.org/WAI/media/av/planning/#checklist> we say “The checklists below … cover … What is required in WCAG… [and] What is needed to meet user needs, beyond WCAG.”

I think need to re-write to better explain that (sorry I don't have a specific suggestion tight now)

**Comment by daniel-montalvo** [comment]
Thanks @shawna-slh 


As per b29cbde76a11a40451b11dbfd735d025c9b46e28

I have moved that sentence to a  separate teaching idea, where I think it fits better based on these comments.

I propose:

> Explain that different government regulations and other policy requirements cover different user needs for audio and video content. Emphasize that some government regulations and policies may not cover all user needs depending on audience and context.

This:

- Points instructors to the lack of harmonization between different policies and regulations.
- Emphasize that it is sometimes needed to go beyond the specific policy to cover user needs.

**Comment by shawna-slh** [comment]
In follow up to our chat on Tuesday:

> Points instructors to the lack of harmonization between different policies and regulations.

Perhaps this is a high-level issue that we don't need to highlight out here?

> Emphasize that it is sometimes needed to go beyond the specific policy to cover user needs.

I would say that you need to go beyond most regulations and policies to cover user needs -- definitely beyond WCAG 2 AA.

> Explain that different government regulations and other policy requirements cover different user needs for audio and video content. Emphasize that some government regulations and policies may not cover all user needs depending on audience and context.

Perhaps something like:

> Explain that some government regulations and policies do not cover all user needs, and content creators should go beyond minimal requirements to meet user needs.

**Comment by daniel-montalvo** [comment]
[Updated](https://content-author-modules--wai-curricula.netlify.app/curricula/content-author-modules/multimedia/#topic-planning-audio-and-video):

> Emphasize that some government regulations and policies do not cover all user needs. Content creators should go beyond minimal requirements to meet all user needs. Mention government regulations and other policy requirements that apply in your students region.

We removed the sentence "Explain that different policies and regulations apply to different regions". But the TF still felt the need to give instructors a pointer to mention policies and regulations that apply in their students region and we have kept that part.
